.env.example

```
@@ -16,13 +16,25 @@ DIRECT_URL=postgresql://postgres:[YOUR-PASSWORD]@db.xxx.supabase.co:5432/postgre
 
 # OpenAI API (for EBM Chat)
 # Get from: https://platform.openai.com/api-keys
+# ChatWell will be unavailable if this is missing or blank
 OPENAI_API_KEY=sk-your-openai-key-here
 
+# Mock AI provider (tests/CI)
+# Set MOCK_AI=true or AI_PROVIDER=mock to disable external API calls (only for local dev/tests; keep false/real provider in prod)
+MOCK_AI=false
+MOCK_AI_RESPONSE=ChatWell (modo de teste): resposta simulada para validação do fluxo.
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)